bundles/org.eclipse.equinox.security.tests/src/org/eclipse/equinox/internal/security/tests/storage/ManualTest.java

```diff
@@ -60,33 +60,21 @@ public void testBasic() {
 			}
 
 			/*
-			 * In theory, all those ciphers support "Raw" key factory format - the SecretKeyFactory
-			 * algorithm should be really irrelevant. That said, Blowfish would not work with JavaCrypt
-			 * - so need to manually test all and exclude those that we know not supported
-			 * (or provide a roundtrip test - if it fails, exclude the value)
-		[keyFactory] PBKDF2WithHmacSHA1
```

Comment:
And I suppose this is the other place.

@vogella Would you like this as an `<ul>`-list, `<ol>`-list or just as a `<pre>`-block?